devito/ir/support/properties.py

```diff
@@ -16,11 +16,17 @@ def __init__(self, name, val=None):
 PARALLEL = Property('parallel')
 """A fully parallel Dimension."""
 
+PARALLEL_INDEP = Property('parallel=')
+"""
+A fully parallel Dimension; parallelism is due to all dependences having
```

Comment:
Nitpicking: ...is due to all dependence distances being/equal/ to 0... I think it is `a dependence distance`, not that a dependence owns a distance....but maybe I am just overcommenting here...

Reply:
fixed. before it was kinda okish in the sense that in Devito, a `Dependence` object has a `distance` property. But now it's better, I agree